pkg/cronjob/es_index_cleaner_test.go

```diff
@@ -16,13 +16,17 @@ func TestCreateEsIndexCleaner(t *testing.T) {
 		map[string]interface{}{"es.index-prefix": "tenant1", "es.server-urls": "http://nowhere:666,foo"})}}}
 	days := 0
 	jaeger.Spec.Storage.EsIndexCleaner.NumberOfDays = &days
+	historyLimits := int32(1)
+	assert.Nil(t, jaeger.Spec.Storage.EsIndexCleaner.SuccessfulJobsHistoryLimit)
```

Comment:
That's not really what I meant, but it just made me realize that the `assert.Nil` that I requested before didn't make much sense either. Could you revert this one?

Reply:
done